<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessionPathsSetsWidget</class>
 <widget class="QWidget" name="SessionPathsSetsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="KButtonGroup" name="favGroup">
     <property name="title">
      <string>Favorite and Stored Sets</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="lh2">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Session Paths Set:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="setsList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hl1">
        <item>
         <spacer name="hs1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="KPushButton" name="storeButton">
          <property name="text">
           <string>Save current set</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="addButton">
          <property name="text">
           <string>Add a set to the list</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>addButton</zorder>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="KButtonGroup" name="suggestionGroup">
     <property name="title">
      <string>Suggestion Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="onlyVCS">
        <property name="text">
         <string>Add only directories with VCS serivice directory</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="onlyCommonRoot">
        <property name="text">
         <string>Add only directories with common root</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="allPaths">
        <property name="text">
         <string>Add all directories</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hl3">
        <item>
         <spacer name="hs2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="KPushButton" name="guessButton">
          <property name="text">
           <string>Suggest a set</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KButtonGroup</class>
   <extends>QGroupBox</extends>
   <header>kbuttongroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
